### PR TITLE
Fix updating stats in uncommitted case in txn-emitter

### PR DESCRIPTION
**Pull request by banool**
## Description
What we were doing previously was totally invalid, we were subtracting a number of accounts from a number of transactions. This fixes that up + adds a comment explaining what is going on. The amount of time we were waiting for transactions in this situation was also needlessly long.

## Test Plan
See https://www.notion.so/aptoslabs/Single-node-testnet-transaction-load-test-guide-57e82a909e614243bed9694641f581f9.

### CODE REVIEW

File: crates/transaction-emitter/src/lib.rs [hunk 1]
```diff
@@ -270,6 +234,69 @@ impl SubmissionWorker {
...
+                    .fetch_add(num_requests as u64, Ordering::Relaxed);
+                self.stats
+                    .latency
+                    .fetch_add(latency * num_requests as u64, Ordering::Relaxed);
+                self.stats
+                    .latencies
+                    .record_data_point(latency, num_requests as u64);
+            }
+            Err(committed_txn_info) => {
+                let num_committed = committed_txn_info
```
Comment (sitalkedia) [inline crates/transaction-emitter/src/lib.rs:264]: This doesn't look right to me. We should only add the committed txns incrementally right?

**Comment by sitalkedia** [comment]
@banool - LMK when you have fixed the committed stat, I will take another look at this.

File: crates/transaction-emitter/src/lib.rs (hunk 1 above)
Comment (banool) [reply crates/transaction-emitter/src/lib.rs:264]: You're right, the original code was legit, I needed to look closer at gen_requests. I'll fix that up now.

File: crates/transaction-emitter/src/lib.rs [hunk 2]
```diff
@@ -526,7 +550,7 @@ impl<'t> TxnEmitter<'t> {
             });
         let mut minted_accounts = try_join_all(account_futures)
             .await
-            .context("Failed to mint accounts")?
+            .map_err(|e| format_err!("Failed to mint accounts: {}", e))?
```
Comment (sitalkedia) [inline crates/transaction-emitter/src/lib.rs:553]: Thank you for fixing the logging.

**Pull request merged**